app/components/post_textbox/post_textbox.js

```diff
@@ -188,9 +189,19 @@ class PostTextbox extends PureComponent {
         }
 
         // Shrink the input textbox since the layout events lag slightly
-        this.setState({
+        const nextState = {
```

Comment:
It's fine to have this like
```
this.setState({
    contentHeight: INITIAL_HEIGHT
});

if (Platform.OS === 'android') {
    this.setState({
        keyboardType: 'email-address'
    }, () => {
        this.setState({
            keyboardType: 'default
        });
    });
}
```
since React batches multiple setState calls.